compiler/ilgen/MethodBuilder.cpp

```diff
@@ -95,7 +96,14 @@ MethodBuilder::MethodBuilder(TR::TypeDictionary *types, OMR::VirtualMachineState
    _allBytecodeBuilders(0),
    _vmState(vmState),
    _bytecodeWorklist(NULL),
-   _bytecodeHasBeenInWorklist(NULL)
+   _bytecodeHasBeenInWorklist(NULL),
+   _parameterSlot(str_comparator),
```

Comment:
This list looks out of order to me; these fields should be listed in order of their definition in the class. Putting them here in a different order is misleading since the fields won't be initialized in this order (not that there are dependencies, but it's cleaner if the order matches the order the fields will be initialized).